framework/console/controllers/MessageController.php

```diff
@@ -436,38 +436,38 @@ protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messag
             $freshMessagesIds[] = $row['id'];
         }
             
-        $this->stdout("Generating missing rows...");
+        $this->stdout('Generating missing rows...');
         $generatedMissingRows = [];
         
         foreach ($languages as $language) {
-          $count = 0;
+            $count = 0;
           
           // get list of ids of translations for this language
-          $msgRowsIds = [];
-          $msgRows = (new Query())->select(['id'])->from($messageTable)->where([
+            $msgRowsIds = [];
+            $msgRows = (new Query())->select(['id'])->from($messageTable)->where([
               'language'=>$language,
-          ])->all($db);
-          foreach ($msgRows as $row) {
-              $msgRowsIds[] = $row['id'];
-          }
+            ])->all($db);
+            foreach ($msgRows as $row) {
+                $msgRowsIds[] = $row['id'];
+            }
           
           // insert missing
-          foreach ($freshMessagesIds as $id) {
-            if (!in_array($id, $msgRowsIds)) {
-              $db->createCommand()
-                 ->insert($messageTable, ['id' => $id, 'language' => $language])
-                 ->execute();
-              $count++;
+            foreach ($freshMessagesIds as $id) {
+                if (!in_array($id, $msgRowsIds)) {
+                    $db->createCommand()
+                     ->insert($messageTable, ['id' => $id, 'language' => $language])
```

Comment:
This is looking worse than it was.

Reply:
Yeah, we need to tweak the standards first to avoid those useless changes.

Reply:
This is just increasing the indent, it was wrong before, see this vscode diff:
<img width="1365" alt="Screenshot 2024-03-03 at 11 42 25" src="https://github.com/yiisoft/yii2/assets/4599319/05ce2d4d-3f9e-4e7f-8e93-c9a2bc18f466">

Reply:
Nope. See this:

<img width="292" alt="image" src="https://github.com/yiisoft/yii2/assets/47294/dbfb9166-6c93-41fa-ad78-15f4fbacaa2c">

It was properly indented but now it's not.

Reply:
Comments have wrong indention too.

Reply:
Fixed this one manually, please check again.